psh/tools/psh.py

```diff
@@ -35,9 +35,20 @@ def _readable(exp_regex):
     return exp_regex.replace(PROMPT, '(psh)% ')
 
 
-def assert_cmd(pexpect_proc, cmd, expected='', msg='', is_regex=False):
+def _check_result(pexpect_proc, result):
+    if result == 'dont-check':
+        return
+
+    if (result != 'success') and (result != 'fail'):
+        raise Exception(f''.join((f'The value {result} for `result` argument is not correct. ',
+            'Please choose between susccess/fail/dont-check')))
+
+    assert_cmd_successed(pexpect_proc) if result == 'success' else assert_cmd_failed(pexpect_proc)
```

Comment:
Use normal if else statement when expression value is not used.

Whole function can be done in sigle if [elif] else statement.